docs/content/integrations/dbt/reference.mdx

```diff
@@ -609,6 +609,41 @@ def my_dbt_assets(context: AssetExecutionContext, dbt: DbtCliResource):
 
 ---
 
+## Generate row count data for dbt models
```

Comment:
Maybe this should be grouped under a section "Customizing asset materialization metadata".

Then, under that section, we should add "Fetch column-level lineage"